packages/modules/manage-versions/src/mocks.ts

```diff
@@ -48,6 +48,7 @@ export const MockedChangeset = (
     displayName: `${index}`,
     description: `ch_description${index}`,
     pushDateTime: MOCKED_DATE,
+    synchronizationInfo: { changedFiles: ["test"] },
```

Comment:
Check in tests that this new column is being shown but maybe append index to the file name and maybe rename it to something like `test-file-${index}.dgn`.

Reply:
change pushed